manifests/server/config.pp

```diff
@@ -33,17 +33,21 @@
 
   file { "${puppet::vardir}/reports":
     ensure => directory,
+    mode   => '0750',
     owner  => $puppet::server_user,
```

Comment:
I wonder if this also needs a group. Puppet doesn't enforce it I think, but can't hurt.